src/OpenTelemetry.Instrumentation.AWSLambda/CHANGELOG.md

```diff
@@ -2,8 +2,13 @@
 
 ## Unreleased
 
-* Change dependency from `OpenTelemetry.Contrib.Extensions.AWSXRay` to `OpenTelemetry.Extensions.AWS`
-  ([#1289](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1289))
+## 1.2.0-beta.1
+
+Released 2023-Aug-03
+
+* BREAKING: `AddAWSLambdaConfigurations` no longer removes all existing resource attributes
```

Comment:
To long lines - please markdown lint check.

Reply:
I know. Fixed already.